admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -573,6 +573,17 @@ var specificPrices = (function() {
   var productPriceField = $('#form_step2_specific_price_sp_price');
   var discountTypeField = $('#form_step2_specific_price_sp_reduction_type');
   var discountTaxField = $('#form_step2_specific_price_sp_reduction_tax');
+  var initSpecificPriceForm = new Object();
+
+  /** Get all specific prices */
+  function getInitSpecificPriceForm() {
+    $('#specific_price_form').find('select,input').each(function () {
+        initSpecificPriceForm[$(this).attr('id')]=$(this).val();
```

Comment:
It's even [part of the JS styleguide we use](https://github.com/airbnb/javascript#whitespace--infix-ops) :)